docs/docs/handlers-and-http/routing.md

```diff
@@ -107,6 +107,29 @@ In the above example, the following URLs would be generated by Marten in additio
 
 As you can see, both the URLs and the route names end up being prefixed respectively with the path and the name specified in the including route.
 
+:::info
+The `name` parameter for included routes is optional, i.e. `path "/articles", Articles::ROUTES` is also valid. Please note that this will increase the possibility of a name collision and it is therefore advisable to prefix the individual paths of the included route,
```

Comment:
Can we put the `#path` call example in a dedicated snippet to improve readability?